recipes/cjson/all/conanfile.py

```diff
@@ -15,11 +15,13 @@ class CjsonConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "cjson_utils": [True, False],
```

Comment:
Would `"utils"` be fine too?

Reply:
I guess.
In the meantime cjson_utils is closer to upstream option name: https://github.com/DaveGamble/cJSON/blob/4100379a04f2f2838ffb00c4ad3d0dd9e49f4147/CMakeLists.txt#L172

Reply:
Yes, but they use the `CJSON_` prefix for everyting:
https://github.com/DaveGamble/cJSON/blob/4100379a04f2f2838ffb00c4ad3d0dd9e49f4147/CMakeLists.txt#L232

Reply:
Actually only for these 2 options, none of others are prefixed. But we can rename this option to `utils`, it's understandable for consumers.